examples/cloudbuild_enabled/cloudbuild-tf-plan.yaml

```diff
@@ -12,10 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-control "gcp" do
-  title "GCP Resources"
-
-  describe google_storage_bucket(name: attribute("bucket_name")) do
-    it { should exist }
-  end
-end
+timeout: 1200s
+steps:
+# terraform init -input=false
+- name: gcr.io/$PROJECT_ID/terraform
+  args:
+  - init
+  - -input=false
```

Comment:
We should include `terraform validate` before `terraform plan`.

Reply:
Yep makes sense, I'll add that